modules/creating-infra-machines.adoc

```diff
@@ -3,10 +3,17 @@
 // * post_installation_configuration/cluster-tasks.adoc
 
 [id="creating-infra-machines_{context}"]
-= Creating infrastructure machines
+= Creating infrastructure machines with a dedicated configuration
 
 If you need infrastructure machines to have dedicated configurations, then you must create an infra pool.
 
+[NOTE]
+====
+You do not need a custom pool unless you change a specific node to have a different set of machine configs applied to it. You do not need a custom machine config pool or machine config for infra nodes to work correctly, as it is not a strict requirement for tasks like node labeling, scheduling specific workloads, isolation, or preventing other workload scheduling.
```

Comment:
Maybe we can make this simpler?... Or more direct to the purpose?

"Optionally, if you would like to remove the worker label from your infrastructure nodes, you must create a MachineConfigPool specific to your infrastructure nodes."

Or, with an explanation to the above?

"Optionally, if you would like to remove the worker label from your infrastructure nodes, you must create a MachineConfigPool specific to your infrastructure nodes, as the worker MCP finds members of its pool by the worker role.  If you remove the role that defines that membership from a node, you must create a new MCP for that node to continued to be controlled by the MCO."